src/Models/NonhydrostaticModels/nonhydrostatic_model.jl

```diff
@@ -50,7 +55,8 @@ end
 """
     NonhydrostaticModel(;     grid,
                               clock = Clock{eltype(grid)}(0, 0, 1),
-                          advection = CenteredSecondOrder(),
+                 momentum_advection = CenteredSecondOrder(),
+                   tracer_advection = CenteredSecondOrder(),
```

Comment:
I prefer advection, and we should do this in a different PR.

Reply:
Ok, then we can change it in a different PR (for the moment then separate `bounds` can be used only in the `HydrostaticFreeSurfaceModel`)